app/assets/stylesheets/era_custom.scss

```diff
@@ -1,3 +1,9 @@
+@import 'avalon';
+
+$link1: #b5fd99;
+$link1Hover: #efefef;
+$link1Visited: #b5fd99;
```

Comment:
Sass Variables should be dashed-cased, instead of camelcase

So `$link1Visited` vs `$link-1-visited`.

Also why name this `1`, what does this refer too? Give this a better name? e.g: `$footer-link` vs `$link1`